docs/src/app/global.css

```
@@ -201,6 +201,45 @@ pre {
+   animations never run (reduced motion, headless renderers, hidden tabs). */
+@media (prefers-reduced-motion: no-preference) {
+  .lp-rise {
+    animation: lp-rise 0.7s cubic-bezier(0.22, 1, 0.36, 1) backwards;
+    animation-delay: calc(var(--lp-rise-delay, 0) * 90ms);
+  }
+  .lp-pulse::before {
+    content: "";
+    position: absolute;
+    inset: 0;
+    border-radius: inherit;
+    background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)